authors/apps/articles/models.py

```diff
@@ -1,47 +0,0 @@
-from django.db import models
-from django.core.validators import MaxValueValidator, MinValueValidator
```

Comment:
Why are you deleting all these files?

Reply:
When i rebased from Develop, I found that Gidraf had created an articles folder and I had article folder without an "s" at the end. In this light I had to merge the work to one folder thus deleting the other folder and all its contents since they were all now intergrated to the article folder.

Reply:
Make sense, was wondering what was happening, I remember this. That all for me.